packages/server/src/integrations/elasticsearch.ts

```diff
@@ -81,7 +99,25 @@ class ElasticSearchIntegration implements IntegrationBase {
 
   constructor(config: ElasticsearchConfig) {
     this.config = config
-    this.client = new Client({ node: config.url })
+
+    let newConfig = { 
+      node: this.config.url,
+      ssl: this.config.ssl 
+        ? {
+          rejectUnauthorized: this.config.rejectUnauthorized,
+          ca: this.config.ca ? this.config.ca : undefined
+        }
+        : undefined,
+    }
+
+    if (newConfig.ssl && !newConfig.ssl.ca)
+    {
```

Comment:
Please wrap the brace on the same line as the statement, e.g.
```
if (newConfig.ssl && !newConfig.ssl.ca) {
...
}
```
same for the *else if* below.

Reply:
Whoops! Got it, I’ve fixed the styling differences.